contract_show_invoice/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Contract Show Invoice
+=====================
+
+This module add a button in contracts to show their invoices.
+
```

Comment:
Maybe we could be more precise. If I understood correctly:

    All invoices including lines for the Contract will be shown.